plugins/mumble_plugin_utils.h

```diff
@@ -18,6 +18,7 @@
 // when constructing JSON documents via
 // string concatenation.
 static void escape(char *str) {
+    str[sizeof(str)-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
```

Comment:
This doesn't do what you expect.

Since str is a pointer to char, not a char array, sizeof(str) will return the size of a pointer for the system, NOT the size of the array in bytes.